card/mod/02_basic_types/set/all/base.rb

```diff
@@ -10,13 +10,23 @@ def show view, args
   view :name, simple_args do |args|
     return card.name unless args[:variant]
     args[:variant].split(/[\s,]+/).inject(card.name) do |name, variant|
-      if ::Set.new([
-        :downcase, :upcase, :singularize, :pluralize, :capitalize,
-        :swapcase, :reverse, :succ
-      ]).include?(variant.to_sym)
-        name.send variant
+      case variant.to_sym
+      when :capitalized
+        name.capitalize
+      when :singular
+        name.singularize
+      when :plural
+        name.pluralize
+      when :title
+        name.titleize
       else
-        name
+        if ::Set.new([
+          :downcase, :upcase, :swapcase, :reverse, :succ
+        ]).include?(variant.to_sym)
+          name.send variant
+        else
+          name
+        end
```

Comment:
the logic is good.  the code complexity...definitely good enough for a ski vacation :)

wondering if rubocop/hound/codeclimate can be configured to treat view and event block complexity the same way it treats method complexity
?